onnxruntime/core/providers/openvino/backends/vadm_backend.cc

```diff
@@ -208,7 +207,7 @@ void VADMBackend::CompleteAsyncInference(Ort::CustomOpApi& ort, OrtKernelContext
 
     FillOutputBlob(graph_output_blob, output_tensor, ort, precision, batch_slice_idx);
   }
-#if defined(OPENVINO_2020_4)
+#if defined(OPENVINO_2021_1) || defined(OPENVINO_2021_2) || defined(OPENVINO_2021_3) || defined(OPENVINO_2021_4)
```

Comment:
same comment as above.
Was this a previous mistake?  was the logic supposed to be 
for 2020.4 and later?

Reply:
Yes George, This is correct. The check had to be there for all the currented supported versions of OpenVINO except OV 2020.3 version. Yes it was a mistake. The logic had to be for 2020.4 and later. when we were doing the refactoring stuff for GetCapability() changes, we should have added it.

once we get rid of 2020.3 LTS support from OpenVINO-EP after our next release (2020.4 LTS). we can just remove those #if defined checks. That condition is must for all the current supported versions.

Reply:
thanks for the explanation!